resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -814,6 +814,326 @@
           label: Pharmacokinetics and pharmacodynamics/Description
           type: basic:string
           required: false
+        - name: mutation_status_kras
+          label: Mutation status/KRAS
+          type: basic:string
+          required: false
+          choices:
+            - label: KRAS G12D
```

Comment:
Are these mutations sorted by the frequency of affected cases? I would sort them alphabetically and/or by amino acid position.